readme.md

```diff
@@ -73,8 +73,7 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 [OpenRCT2.org](https://openrct2.org/downloads) offers precompiled builds and installers of the latest master and the develop branch. There is also a cross platform [Launcher](https://github.com/LRFLEW/OpenRCT2Launcher/releases) available that will automatically update your build of the game so that you always have the latest version.
 
 [Flathub](https://flathub.org/) offers flatpaks for Linux distributions that support this application distribution system:
-* [Latest stable release](https://flathub.org/repo/appstream/io.openrct2.OpenRCT2.flatpakref)
-* [Latest development build](https://flathub.org/beta-repo/appstream/io.openrct2.OpenRCT2.flatpakref)
+* [Latest stable release](https://flathub.org/apps/details/io.openrct2.OpenRCT2)
```

Comment:
```suggestion
* [Latest release](https://flathub.org/apps/details/io.openrct2.OpenRCT2)
```

We don’t call our releases stable, they’re just releases.

Reply:
Changed.